src/pal/tests/palsuite/runpaltests.sh

```diff
@@ -52,6 +52,16 @@ echo PAL tests will store their temporary files and output in $PAL_TEST_OUTPUT_D
 # Path to a file that will contains a list PAL tests that failed during the test run.
 PAL_FAILED_TEST_LIST=$PAL_TEST_OUTPUT_DIR/palfailedtests.txt
 
+# Path to a file that will contain the JUnit style test result for Jenkins
+# We use a temp file as at the end we have to prepend with the number of tests
+# and failures
+PAL_JUNIT_TEST_LIST_TMP=$PAL_TEST_OUTPUT_DIR/pal_tests.xml.tmp
+PAL_JUNIT_TEST_LIST=$PAL_TEST_OUTPUT_DIR/pal_tests.xml
+
```

Comment:
@benpye, "JUnit is a unit testing framework for the Java programming language." while "xunit is a unit testing framework for C# and .NET". I don't understand the reason why would you use JUnit terminology in ".NET Core" repository, when xunit is equally supported by Jenkins. Is it a pun :trollface: ?